js/interface.js

```diff
@@ -1,30 +1,30 @@
-var defaultData = {
+const defaultData = {
```

Comment:
this woul break on IE11

Reply:
what linting are you using? doesn't look like it's the one in use by ES5 components

Reply:
@squallstar Yeh I just realised this. It's really weird. It's automatically turning everything ES6. Let me take a look.